README.md

```diff
@@ -991,6 +991,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [Cachet](https://cachethq.io/) - An open source status page system for everyone. ([Demo](https://demo.cachethq.io/), [Source Code](https://github.com/CachetHQ/Cachet)) `BSD-3-Clause` `PHP`
 - [CUPS](https://www.cups.org/) - The Common Unix Print System uses Internet Printing Protocol (IPP) to support printing to local and network printers. ([Source Code](https://github.com/apple/cups)) `GPL-2.0` `C`
 - [CyberChef](https://github.com/gchq/CyberChef) - Perform all manner of "cyber" operations within a web browser such as AES, DES and Blowfish encryption and decryption, creating hexdumps, calculating hashes, and much more. ([Demo](https://gchq.github.io/CyberChef)) `Apache-2.0` `JavaScript`
+- [Digital-Currency](https://github.com/Icesofty/Digital-Currency#readme) - Create your own Self-Hosted Digital Currency. ([Demo](http://tonken.mooo.com/), [Source Code](https://github.com/Icesofty/Digital-Currency)) `GPL-3.0` `Nodejs`
```

Comment:
Remove #readme from the end of the main link.
Remove the Source code link as it is a duplicate of the main link.

Reply:
Roger that